tldw_Server_API/app/api/v1/endpoints/media/process_documents.py

```
@@ -478,6 +503,53 @@ async def _document_batch_processor(
+                        if isinstance(
+                            chunk_options_dict.get("hierarchical_template"), dict
+                        )
+                        else None,
+                    )
+                else:
+                    chunks = _improved_chunking_process(text, chunk_options_dict)
+
+                res["chunks"] = chunks
+    except Exception:
+        # Re-chunking is best-effort; never fail the endpoint if it errors.
+        pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)